docs/Events/bpmn-events/detach.md

```diff
@@ -4,9 +4,25 @@ title: detach
 nav_order: 3
 description: detach event
 permalink: /events/bpmn-events/detach
-last_modified_date: 2020-04-27T17:54:08+0000
+last_modified_date: 2020-09-22T12:00:00+0000
 parent: "BPMN Events"
 grand_parent: "Events"
 ---
 
 # detach
+
+This event is fired by [detach API](/general/api-reference/bpmn-js/detach) after the bpmn-js container was removed from the [configured DOM node](/general/getting-started/bpmn-js). 
```

Comment:
My first comment goes here too: `was removed`? or `is removed` or `gets removed`.